[01-Jul-2018 14:14:52 UTC] PHP Parse error:  syntax error, unexpected 'instaceof' (T_STRING) in /home/transpez/public_html/api/app/Exceptions/Handler.php on line 48
[01-Jul-2018 14:14:52 UTC] PHP Fatal error:  Exception thrown without a stack frame in Unknown on line 0
[01-Jul-2018 14:14:52 UTC] PHP Fatal error:  Exception thrown without a stack frame in Unknown on line 0
[01-Jul-2018 14:15:02 UTC] PHP Parse error:  syntax error, unexpected 'instaceof' (T_STRING) in /home/transpez/public_html/api/app/Exceptions/Handler.php on line 48
[01-Jul-2018 14:15:02 UTC] PHP Fatal error:  Exception thrown without a stack frame in Unknown on line 0
[01-Jul-2018 14:15:02 UTC] PHP Fatal error:  Exception thrown without a stack frame in Unknown on line 0
[01-Jul-2018 14:15:16 UTC] PHP Parse error:  syntax error, unexpected 'instaceof' (T_STRING) in /home/transpez/public_html/api/app/Exceptions/Handler.php on line 48
[01-Jul-2018 14:15:16 UTC] PHP Fatal error:  Exception thrown without a stack frame in Unknown on line 0
[01-Jul-2018 14:15:16 UTC] PHP Parse error:  syntax error, unexpected 'instaceof' (T_STRING) in /home/transpez/public_html/api/app/Exceptions/Handler.php on line 48
[01-Jul-2018 14:15:16 UTC] PHP Fatal error:  Exception thrown without a stack frame in Unknown on line 0
[01-Jul-2018 14:22:37 UTC] PHP Fatal error:  Uncaught ErrorException: Undefined variable: e in /home/transpez/public_html/api/app/Exceptions/Handler.php:48
Stack trace:
#0 /home/transpez/public_html/api/app/Exceptions/Handler.php(48): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Undefined varia...', '/home/transpez/...', 48, Array)
#1 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(112): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(ErrorException))
#2 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(89): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(ErrorException))
#3 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(ErrorException))
#4 {main}
  thrown in /home/transpez/public_html/api/app/Exceptions/Handler.php on line 48
[01-Jul-2018 14:24:45 UTC] PHP Fatal error:  Uncaught ErrorException: Undefined variable: e in /home/transpez/public_html/api/app/Exceptions/Handler.php:48
Stack trace:
#0 /home/transpez/public_html/api/app/Exceptions/Handler.php(48): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Undefined varia...', '/home/transpez/...', 48, Array)
#1 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(112): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(ErrorException))
#2 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(89): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(ErrorException))
#3 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(ErrorException))
#4 {main}
  thrown in /home/transpez/public_html/api/app/Exceptions/Handler.php on line 48
[01-Jul-2018 14:30:54 UTC] PHP Fatal error:  Uncaught ErrorException: Undefined variable: e in /home/transpez/public_html/api/app/Exceptions/Handler.php:49
Stack trace:
#0 /home/transpez/public_html/api/app/Exceptions/Handler.php(49): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Undefined varia...', '/home/transpez/...', 49, Array)
#1 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(112): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(ErrorException))
#2 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(89): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(ErrorException))
#3 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(ErrorException))
#4 {main}
  thrown in /home/transpez/public_html/api/app/Exceptions/Handler.php on line 49
[01-Jul-2018 14:58:29 UTC] PHP Fatal error:  Uncaught Error: Call to a member function send() on null in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php:112
Stack trace:
#0 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(89): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Symfony\Component\Debug\Exception\FatalThrowableError))
#1 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\Debug\Exception\FatalThrowableError))
#2 {main}
  thrown in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php on line 112
[01-Jul-2018 14:58:29 UTC] PHP Fatal error:  Uncaught Error: Call to a member function send() on null in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php:112
Stack trace:
#0 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(89): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Symfony\Component\Debug\Exception\FatalErrorException))
#1 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(123): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\Debug\Exception\FatalErrorException))
#2 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 {main}
  thrown in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php on line 112
[01-Jul-2018 14:59:00 UTC] PHP Fatal error:  Uncaught Error: Call to a member function send() on null in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php:112
Stack trace:
#0 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(89): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Symfony\Component\Debug\Exception\FatalThrowableError))
#1 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\Debug\Exception\FatalThrowableError))
#2 {main}
  thrown in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php on line 112
[01-Jul-2018 14:59:00 UTC] PHP Fatal error:  Uncaught Error: Call to a member function send() on null in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php:112
Stack trace:
#0 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(89): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Symfony\Component\Debug\Exception\FatalErrorException))
#1 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(123): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\Debug\Exception\FatalErrorException))
#2 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 {main}
  thrown in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php on line 112
[01-Jul-2018 14:59:01 UTC] PHP Fatal error:  Uncaught Error: Call to a member function send() on null in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php:112
Stack trace:
#0 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(89): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Symfony\Component\Debug\Exception\FatalThrowableError))
#1 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\Debug\Exception\FatalThrowableError))
#2 {main}
  thrown in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php on line 112
[01-Jul-2018 14:59:01 UTC] PHP Fatal error:  Uncaught Error: Call to a member function send() on null in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php:112
Stack trace:
#0 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(89): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Symfony\Component\Debug\Exception\FatalErrorException))
#1 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(123): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\Debug\Exception\FatalErrorException))
#2 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 {main}
  thrown in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php on line 112
[01-Jul-2018 17:05:02 UTC] PHP Fatal error:  Uncaught Error: Call to a member function send() on null in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php:112
Stack trace:
#0 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(89): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Symfony\Component\Debug\Exception\FatalThrowableError))
#1 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\Debug\Exception\FatalThrowableError))
#2 {main}
  thrown in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php on line 112
[02-Jul-2018 02:34:20 UTC] PHP Fatal error:  Uncaught Error: Call to a member function send() on null in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php:112
Stack trace:
#0 /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(89): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Symfony\Component\Debug\Exception\FatalThrowableError))
#1 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\Debug\Exception\FatalThrowableError))
#2 {main}
  thrown in /home/transpez/public_html/api/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php on line 112
[02-Jul-2018 02:37:00 UTC] PHP Fatal error:  Cannot redeclare App\Exceptions\Handler::unauthenticated() in /home/transpez/public_html/api/app/Exceptions/Handler.php on line 84
[02-Jul-2018 02:37:05 UTC] PHP Fatal error:  Cannot redeclare App\Exceptions\Handler::unauthenticated() in /home/transpez/public_html/api/app/Exceptions/Handler.php on line 84
[02-Jul-2018 02:38:54 UTC] PHP Parse error:  syntax error, unexpected 'if' (T_IF), expecting function (T_FUNCTION) or const (T_CONST) in /home/transpez/public_html/api/app/Exceptions/Handler.php on line 49
[02-Jul-2018 02:38:54 UTC] PHP Fatal error:  Exception thrown without a stack frame in Unknown on line 0
[02-Jul-2018 02:38:54 UTC] PHP Fatal error:  Exception thrown without a stack frame in Unknown on line 0
